### PR TITLE
[INF-353] Add vector.dev container

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add support for vector.dev based on https://github.com/AudiusProject/audius-protocol/pull/4964

Add `remove-orphans` to each compose up command so that containers no longer specified in the compose will be cleaned up on the next launch.

While technically this can be merged at any time, I would like to get this on all staging instances first for an extended period.
Is there a best way to do this? Maybe just set the branches?

### CODE REVIEW

**Review by rickyrombo — APPROVED** [review]
I think there used to be a reservation system, but honestly the easiest way is probably just to merge it and revert it on the release branch?

**Comment by raymondjacobson** [comment]
> I think there used to be a reservation system, but honestly the easiest way is probably just to merge it and revert it on the release branch?

this is audius-docker-compose, not protocol, but yeah that's the right idea

**Comment by cheran-senthil** [comment]
Best is to change branch on the stage nodes that you want to consume this by hand.

Can build a flow if you'd like to click a circle button / run a bash script instead. Think that might be a good idea anyway

File: audius-cli [hunk 1]
```diff
@@ -304,7 +304,8 @@ def launch(ctx, service, seed, chain, yes):
             "--project-directory",
             ctx.obj["manifests_path"] / service,
             "up",
-            "-d",
+            "--remove-orphans",
```
Comment (cheran-senthil) [inline audius-cli:307]: Nice

File: common-services.yml [hunk 2]
```diff
@@ -38,12 +38,12 @@ services:
...
-  logspout:
-    image: audius/logspout:v3.2.14-1
-    container_name: logspout
+  vector:
+    image: audius/vector:0.28.1-alpine
+    container_name: vector
     restart: always
     volumes:
-      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - /var/run/docker.sock:/var/run/docker.sock
```
Comment (cheran-senthil) [inline common-services.yml:46]: This is very safe to merge, and unlikely to cause any unintended effects

**Review by cheran-senthil — APPROVED** [review]
LGTM!

**Comment by cheran-senthil** [comment]
btw, as long as this branch stays up, auto deploy to stage will still work since that relies on git pull

**Pull request merged**